src/scripts/main.js

```diff
@@ -1,3 +1,135 @@
 'use strict';
 
-// write code here
+// sort function
+const tHeadTags = document.querySelectorAll('th'); // Вибираємо всі заголовки
+const tbody = document.querySelector('tbody'); // Тіло таблиці
+const rows = tbody.querySelectorAll('tr'); // Всі рядки
+
+const sortMode = {}; // Об'єкт для відстеження стану сортування
+
+tHeadTags.forEach((tag, thIndex) => {
+  // Початковий стан сортування — за зростанням
+  sortMode[thIndex] = 'asc';
+
+  tag.addEventListener('click', () => {
+    const direction = sortMode[thIndex];
+
+    if (direction === 'asc') {
+      sortTable(thIndex, 'asc');
+      sortMode[thIndex] = 'desc'; // Змінюємо стан
+    } else {
+      sortTable(thIndex, 'desc');
+      sortMode[thIndex] = 'asc'; // Змінюємо стан
+    }
+  });
+});
+
+function sortTable(indx, mode) {
+  const table = document.querySelector('table');
+  const content = Array.from(table.rows).slice(1, -1); // Пропускаємо заголовки
```

Comment:
The `content` array should include all rows except the header. The current slicing might exclude the last row if there's a footer. Ensure the slicing logic correctly targets only the rows to be sorted.